### DISCUSSION TITLE
Extracting weekdays from header component

### DISCUSSION THREAD

**Discussion by KonradLinkowski** [gpbl/react-day-picker#1532] [Ideas]
Adding anything to the `<thead>` feels like a chore currently.
For example, adding a simple `today` button requires:
- copying the entire content of [`Head`](https://github.com/gpbl/react-day-picker/blob/50b6dba/packages/react-day-picker/src/components/Head/Head.tsx#L10)
- copying the implementation of [`getWeekdays`](https://github.com/gpbl/react-day-picker/blob/50b6dbafd60386eca895cce439a5a014f9c7016b/packages/react-day-picker/src/components/Head/utils/getWeekdays.ts#L10)

Extracting the weekdays' row to a separate component would allow using it like this:

```tsx
const NewHead = () => {
  const {
    classNames,
    styles,
  } = useDayPicker();
  
  return (
    <thead>
      {/* additional content here */}
      <WeekdaysRow />
    </head>
  )
}

<DayPicker
  components={{ Head: NewHead }}
/>
```

Is there already a way of doing this without copying the entire `Head` component?

**Comment by gpbl** [comment]
@KonradLinkowski this is a good idea! Would you mind to propose a PR with this change?

**Reply by KonradLinkowski** [reply]
I created a PR https://github.com/gpbl/react-day-picker/pull/1533, let me know if there is anything that should be improved
